h/static/scripts/widget-controller.js

```diff
@@ -312,9 +347,11 @@ module.exports = function WidgetController(
     streamer.reconnect();
   });
 
-  // When a direct-linked annotation is successfully anchored in the page,
-  // focus and scroll to it
   $rootScope.$on(events.ANNOTATIONS_SYNCED, function (event, tags) {
+    annotationUI.selectTab(uiConstants.TAB_ANNOTATIONS);
+
+    // When a direct-linked annotation is successfully anchored in the page,
+    // focus and scroll to it
```

Comment:
OK, we now have two cases in which this function could be called that are not accounted for by this comment.

The `ANNOTATIONS_SYNCED` event does not actually mean that the annotation is successfully anchored in the page as this comment says, but rather than attempted anchoring is completed, and we now know whether it was anchored or not.